plugin/modelgen/models_test.go

```diff
@@ -298,6 +298,24 @@ func TestModelGeneration(t *testing.T) {
 	})
 }
 
+func TestModelGenerationOmitRootModels(t *testing.T) {
+	cfg, err := config.LoadConfig("testdata/gqlgen_omit_root_models.yml")
+	require.NoError(t, err)
+	require.NoError(t, cfg.Init())
+	p := Plugin{
+		MutateHook: mutateHook,
+		FieldHook:  DefaultFieldMutateHook,
+	}
+	//cfg.OmitRootModels = true
```

Comment:
Can you please either remove this line or add a space to the comment?

Reply:
Oops, this was test code that should have been removed.  Updated!